examples/js/objects/Reflector.js

```diff
@@ -168,6 +168,8 @@ THREE.Reflector = function ( geometry, options ) {
 
 		renderer.setRenderTarget( currentRenderTarget );
 
+		renderer.setupLights( camera ); // rebuild light uniforms, see #12883

```

Comment:
Yes, maybe there is a better solution... In any event, I'd move this line to the line following the call to `render()` where the damage is done.

Reply:
Good idea!